src/main/java/utils/JSActions.java

```diff
@@ -0,0 +1,19 @@
+package utils;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class JSActions {
+
+    private WebDriver driver;
+
+    public JSActions(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public static void clickButtonWithJS(WebDriver driver, WebElement element) {
```

Comment:
You already have WebDriver in this class, there is no need to pass it here.

Reply:
Updated